src/app/tests/suites/certification/Test_TC_DEMM_1_2.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: 241.1.2. [TC-DEMM-1.2] Cluster attributes with DUT as Server
+
+PICS:
+    - DEMM.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 1
+
+tests:
+    - label: "Step 1: TH reads from the DUT the SupportedModes attribute"
```

Comment:
This is in a PR that says automated/semi-automated, but this looks like a fully manual test.